features/richtext.feature

```diff
@@ -14,14 +14,24 @@ Feature: Define dimension using bag.is/is_not_richtext()
     {<P6 'All Work'>, <C6 'New Housing'>, <K6 'Repair and Maintenance'>, <J6 'All New Work'>, <E6 'Total Housing'>, <O6 'All Repair and Maintenance'>, <F6 'Other New Work'>}
     """
 
-Scenario: Define construction type by richtext values in row "6"
-    Given we load an xls file named "bulletindataset2v2.xls"
-    And select the sheet "Table 2a"
+    Examples: File Types
+            | File Name                   |
+            | "bakingtestdataset.xls"     |
+            #| "bakingtestdataset.xlsx"    |
```

Comment:
is this a property thats not implemented? If so it's probably two tests.

test1 - works as _expected_ with xls
test2 - raises NotImplementedError error _as expected_ with xlsx

capturing the code is doing what's expected is more important than the symmetry here.